tendrl/node_agent/ceph_integration/flows/import_cluster.py

```diff
@@ -27,8 +30,14 @@ def run(self):
                                            json.dumps(job))
         if self.node_id in node_list:
             self.parameters['fqdn'] = socket.getfqdn()
-            ceph = "git+https://github.com/Tendrl/ceph_integration.git@v1.0"
-            self.parameters['Package.name'] = ceph
+            if config.get("node_agent", "deployment_type") == "production":
+                self.parameters['Package.name'] = "tendrl-ceph-integration"
+                self.parameters['Package.pkg_type'] = "yum"
```

Comment:
we need to detect rpm or deb support based on node_agent "node" or "system" objects